content/accessibility/accessibility-at-github.mdx

```diff
@@ -0,0 +1,48 @@
+---
+title: Accessibility at GitHub
+---
+
+GitHub's aim is to be the home for *all* developers. To be inclusive means we must consider accessibility at the core of how we design. 
+
+15% of the world's population has a disability (estimated by the [World Health Organization based on 2010 data](https://www.who.int/disabilities/world_report/2011/report/en/)), and that doesn't take into consideration those under temporary or situational disabilities. 
+
+In many cases, devices that have been designed with accessibility in mind for a portion of the population end up benefiting everyone. Take the [OXO brand's Good Grips line of products](https://www.oxo.com/blog/behind-the-scenes/behind-design-oxos-iconic-good-grips-handles/); although they were developed for people who have arthritis, most people found them easier to use than other product lines.
+
+When you consider accessibility from the beginning of the design process and incorporate it throughout, it becomes part of an established design process that seeks to include everyone, of all abilities, and allows everyone to achieve more.
+
+**Note:** When reading through accessibility information, you might find the common abbreviation: "a11y". A11y is a shortening of the word "accessibility". It starts with the first letter, "a", uses the number 11 for the eleven characters inbetween the first and last letter, and ends with the last letter, "y".
```

Comment:
```suggestion
<Note>When reading through accessibility information, you might find the common abbreviation: "a11y". A11y is a shortening of the word "accessibility". It starts with the first letter, "a", uses the number 11 for the eleven characters inbetween the first and last letter, and ends with the last letter, "y".</Note>
```

I believe we can use the [`Note`](https://primer.style/doctocat/components/note) component available in Doctocat for this.